Rakefile

```diff
@@ -78,12 +78,30 @@ namespace :suite do
   end
 end
 
-desc 'Build the Windows gRPC DLLs for Ruby'
-task 'dlls' do
+desc 'Build the Windows gRPC DLLs for Ruby. Optionally one can pass argument to build only dll for given platform.'
+task 'dlls', [:plat] do |t, args|
   grpc_config = ENV['GRPC_CONFIG'] || 'opt'
   verbose = ENV['V'] || '0'
   # use env variable to set artifact build paralellism
   nproc_override = ENV['GRPC_RUBY_BUILD_PROCS'] || `nproc`.strip
+  selected_plat = "#{args[:plat]}"
+
+  w64 = { cross: 'x86_64-w64-mingw32', out: 'grpc_c.64.ruby', platform: 'x64-mingw32' }
```

Comment:
nit: I think we can clean this up by allowing optional `plat` argument to be a list

We can then remove `selected_plat` variable and do something like this to generate build_configs

```
[w64, w32].each do |config|
  if config[:platform] in args[:plat]
    build_configs.append(config)
  end
end
```

Reply:
done.